WORKSPACE

```diff
@@ -463,8 +463,9 @@ go_repository(
 )
 
 # This SHA is obtained from istio/api
-ISTIO_API = "26e98ec08eeaa97ed0bbcdc28fabd79199256073"
-
+#ISTIO_API = "26e98ec08eeaa97ed0bbcdc28fabd79199256073"
+ISTIO_API = "2e6abc1cfc19b5b7dea7a36c8f59ff41381aeead"
```

Comment:
Changes in this file wont be commited, waiting for https://github.com/istio/api/pull/209